cdc/cdc.cc

```diff
@@ -357,13 +386,23 @@ class transformer final {
 
                         values[0] = data_type_for<column_op_native_type>()->decompose(data_value(static_cast<column_op_native_type>(op)));
                         res.set_cell(log_ck, *dst, atomic_cell::make_live(*dst->type, _time.timestamp(), tuple_type_impl::build_value(values)));
+
+                        if (pirow && pirow->has(cdef.name_as_text())) {
+                            values[0] = data_type_for<column_op_native_type>()->decompose(data_value(static_cast<column_op_native_type>(column_op::set)));
```

Comment:
Please add FIXME so that we remember to add support for column tombstones here as well.

Reply:
Uhm, if there is a tombstone here, we will not get a value back, simple as that.

Reply:
Right. Sorry I got confused.